### ISSUE TITLE
The Gatsby OSS team is on holiday!

### ISSUE THREAD

**Issue by sidharthachatterjee** [gatsbyjs/gatsby#20247]
**_TL;DR Gatsby OSS team is on holiday, expect a delayed response_**

Hey!

If you're reading this, you _probably_ need help with a Gatsby site. Or maybe you want to request a feature or report a bug. Or perhaps you had a fantastic new idea and want to discuss it. 

Go ahead an open an issue or PR as usual. 

_However_, please note that most of the Gatsby OSS team is on holiday for the rest of the year. We're going to be spending time with our families, eating plenty, travelling, skiing. Some of us might even get to play Santa 🤶

Please expect a delayed response to your issues, PRs and messages on Discord. 

Happy holidays from all of us! See you in 2020 💜

**Comment by LekoArts** [comment]
@erezrokah @monsij Please don't unpin the issue. Thanks!

**Comment by sidharthachatterjee** [comment]
We're back, folks! 🤗
